docs/articles_en/documentation/openvino-ir-format/operation-sets/operation-specs/signals/istft-16.rst

```diff
@@ -0,0 +1,206 @@
+
+**Short description**: *ISTFT* operation performs Inverse Short-Time Fourier Transform (complex-to-real).
+
+**Detailed description**: *ISTFT* performs Inverse Short-Time Fourier Transform of complex-valued input tensor 
+of shape ``[fft_results, frames, 2]`` or ``[batch, fft_results, frames, 2]``, where:
+
+  * ``batch`` is a batch size dimension
+  * ``frames`` is a number of frames calculated as ``((signal_length - frame_size) / frame_step) + 1`` of the original signal if not centered, or ``(signal_length / frame_step) + 1`` otherwise.
+  * ``fft_results`` is a number calculated as ``(frame_size / 2) + 1`` of the original signal
+  * ``2`` is the last dimension for complex value represented by floating-point values pair (real and imaginary part accordingly)
+
+The output is a restored real-valued signal in a discrete time domain. The shape of the output is 1D ``[signal_length]`` or 2D ``[batch, signal_length]``.
+If the ``signal_length`` is not provided as an input value, it is calculated according to the following rules:
+
+  * ``signal_length = (frames - 1) * frame_step`` for ``center == true`` 
+  * ``signal_length = (frames - 1) * frame_step + frame_size`` for ``center == false`` 
+
+If the ``window_length`` can not be larger than ``frame_size``, but if smaller the window values will be padded with zeros on the left and right side. The size of the left padding is calculated as ``(frame_size - window_length) // 2``, then right padding size is filled to match the ``frame_size``.  
+
+**Attributes**:
+
+* *center*
+
+  * **Description**: Flag that indicates whether padding has been applied to the original signal. It affects output shape, if the ``signal_length`` input is not provided.
+  * **Range of values**:
+
+    * ``false`` - padding has not been applied, default signal length is calculated as ``(frames - 1) * frame_step + frame_size``
+    * ``true`` - padding has been applied, default signal length is calculated as ``(frames - 1) * frame_step``
+  * **Type**: ``boolean``
+  * **Required**: *yes*
+
+* *normalized*
+
+  * **Description**: Flag that indicates wheather the input has been normalized. It is needed to correctly restore the signal and denormalize the output. Output of the STFT is divided by ``sqrt(frame_size)``, when normalized.
+  * **Range of values**:
+
+    * ``false`` - input has not been normalized
+    * ``true`` - input has been normalized
+  * **Type**: ``boolean``
+  * **Required**: *yes*
+
+
+**Inputs**
+
+* **1**: ``data`` - Tensor of type *T*, the ISTFT data input (compatible with a result of STFT operation). **Required.**
+  * The data input shape can be 3D ``[fft_results, frames, 2]`` or 4D ``[batch, fft_results, frames, 2]``.
+* **2**: ``window`` - Tensor of type *T* and 1D shape ``[window_length]``, specifying the window values applied to restore the signal. If the ``window_length`` is smaller than ``frame_size``, window will be padded with zeros on the left and rigt sides. **Required.**
```

Comment:
```suggestion
* **2**: ``window`` - Tensor of type *T* and 1D shape ``[window_length]``, specifying the window values applied to restore the signal. If the ``window_length`` is smaller than ``frame_size``, window will be padded with zeros on the left and right sides. **Required.**
```

Reply:
I know the info about what happens when `window_size` is larger than `frame_size`, but maybe it also should be mentioned here?